packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -33,6 +39,9 @@ import * as os from 'os';
 
 const OPENTELEMETRY_TASK = 'opentelemetry_task';
 const OPENTELEMETRY_TASK_DESCRIPTION = 'OpenTelemetry task identifier';
+const AWS_REGION_VALUE_PREFIX = 'aws:';
+const GCP_GCE_INSTANCE = 'gce_instance';
+const AWS_EC2_INSTANCE = 'aws_ec2_instance';
```

Comment:
Optional: Given that these strings represent the actual monitored resource types, I'm curious your thoughts on putting them in an enum for reuse throughout the monitoring codebase.